### PR TITLE
Improve settings page's user interface

### PR DESCRIPTION
**Pull request by anirudhjain75**
Some changes have been made to the looks in Settings page

simulator screen shot - iphone 6 - 2018-02-25 at 11 34 37

### CODE REVIEW

File: src/settings/SettingsCard.js [hunk 1]
```diff
@@ -12,16 +12,28 @@ import LogoutButton from '../account-info/LogoutButton';
...
-    height: 16,
+    height: 10,
   },
   padding: {
     padding: 16,
   },
   accountButtons: {
-    flex: 1,
-    alignItems: 'flex-end',
+    position: 'absolute',
```
Comment (jainkuniya) [inline src/settings/SettingsCard.js]: Avoid using `absolute` positions.

Comment (anirudhjain75) [reply src/settings/SettingsCard.js]: Hmm OKay

**Comment by gnprice** [comment]
Thanks @anirudhjain75 ! Closing to match your helpful expectation-setting on #1959 that you won't be working on this for now.

For @anirudhjain75 and anyone else pursuing this issue: one thing that would help make it easier to merge and to help us converge on a high-quality result would be to identify in the commit message what the changes are, and where there are several independent changes to break the PR up into separate commits for each change. Details in this doc: https://zulip.readthedocs.io/en/latest/contributing/version-control.html

**Comment by borisyankov** [comment]
@anirudhjain75 I hope this was educational for you. Not every PR gets merged, but every PR leads to learning. 👍

**Pull request closed without merging**